modules/disabling-redirect-private-storage-endpoint-azure.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-redirect-private-storage-endpoint-azure"]
+= Optional: Disabling redirect when using a private storage endpoint on Azure
+
+By default, redirect is enabled when using the image registry with a private storage endpoint on Azure. This ensures that users within the same cluster can pull images from the registry. 
```

Comment:
Redirect is always enabled by default, not only when using a private storage endpoint on Azure. Up to you if we should keep the phrasing tho.

Reply:
> This ensures that users within the same cluster can pull images from the registry.

Hmm this isn't quite true, redirect allows off-loading of traffic from the registry pods into the registry. This should make pulls faster.

Reply:
@flavianmissi sorry, below says spot on, and here it looks like you're giving suggestions :D How about this?

```suggestion
By default, redirect is enabled when using the image registry. Redirect allows off-loading of traffic from the registry pods into the registry, which makes pull faster. 

When redirect is enabled, users from outside of the cluster are unable to pull images from the registry. In some cases, users might want to disable redirect so that users from outside of the cluster can pull images from the registry. 

Use the following procedure to disable redirect.

```

Reply:
looks good! the "spot on" comment was for the paragraph below "When redirect is enabled..." sorry for the confusion!